articles/libraries/lock/v9/types-of-applications.md

```diff
@@ -10,7 +10,7 @@ You can use Auth0 Lock with both Single Page Apps and with Regular WebApps. In t
 
 ## Single Page App
 
-When you're using a Single Page App, you need that after a successful login through Auth0, you get the `id_token` ([JWT](/jwt)) with which you can then call your API. When you want to use [this workflow](/sequence-diagrams), the `responseType` is then a `token`.
+When you're using a Single Page App, you may need an `access_token`, with which you can call your API, and an `id_token`, which contains user profile attributes like the user's name, email, and so forth, typically used for UI display. When you want to use [this workflow](/sequence-diagrams), the `responseType` should be `id_token`.

```

Comment:
I'm thinking we should undo changes to lock9 docs since lock9 does not support api authorization (e.g. `response_type:id_token` is not supported)